api/src/main/java/org/hawkular/inventory/api/MultipleEntityBrowser.java

```diff
@@ -23,7 +23,7 @@
  * @author Lukas Krejci
  * @since 1.0
  */
-interface MultipleEntityBrowser<Entity> {
+public interface MultipleEntityBrowser<Entity> {
```

Comment:
On Monday, February 23, 2015 06:16:57 Jirka Kremser wrote:
> > @@ -23,7 +23,7 @@
> > 
> >   * @author Lukas Krejci
> >   * @since 1.0
> >   */
> > 
> > -interface MultipleEntityBrowser<Entity> {
> > +public interface MultipleEntityBrowser<Entity> {
> 
> It allows for more general form of use. Types are good, but something more
> abstract what returns the entitites might be useful also for the API
> itself. I agree, the word `Browser` might expose some internal terminology,
> what about something more general name like `HasEntities` with a single
> method `entities()` ?
> 

Well, `HasEntities` is not the precise term to use here, because the method 
really just "materializes" the inventory traversal into a set of entities.

Trying hard to think of a meaningful name but I've failed so far.

> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/hawkular/hawkular-inventory/pull/20/files#r25166199